README.md

```diff
@@ -31,7 +31,7 @@ Edit `.env` and setup your own config
 
 ```bash
 # Run a local ganache in one tab
-npx ganache-cli -h 0.0.0.0
+npx ganache-cli -h 0.0.0.0 --gasLimit 8e6
```

Comment:
weird you need this. Do you know why you need it and I don't? I have `v6.8.1`. This should be Ganache specific, not linux specific.